CHANGELOG.md

```diff
@@ -10,14 +10,30 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 ## Unreleased
 
+## 0.5.0
+
 ### Added
 
-- New `scene-ui` example demonstrating a `VelloScene` attached to a `bevy::ui::Node`
+- New `scene_ui` example demonstrating a `VelloScene` attached to a `bevy::ui::Node`
```

Comment:
Line is missing a period at the end (based on the period being present in most other lines).